pages/en/posts/easy-to-build-monetize-and-discover-list-your-web-app-on-google-play.md

```diff
@@ -0,0 +1,49 @@
+---
+title: 'Easy to build, monetize, and discover: List your web app on Google Play'
+metadesc: Build Android apps for devices other than mobile to reach more users.
+tags:
+  - product news
+  - web
+  - trend
+authors:
+  - pjmclachlan
+  - tbuckley
+date: 2020-12-08
+---
+
+_This article originally appeared on the Chromium Blog._
+
+App developers should be able to make money from their creations, whether via ads, purchases, or subscriptions. The first step to successfully monetizing is getting your app discovered.
+
+Now that Progressive Web Apps (PWAs) can be listed in Google Play, we’re excited to announce that web developers can use Google Play payments in their PWA on Chromebooks and Android devices. This makes it even easier to get your PWA in front of more users and start accepting simple, secure payments by listing in Google Play.
+
+## A new way to connect with Chromebook users
+
+Since their release, [Chromebooks and Chrome OS](https://chromeos.dev/en) have proven that a device built around the web can make computing easier, faster, and more secure. Last year, we expanded high-quality app experiences beyond mobile by [adding support for PWAs on Chrome OS](https://www.youtube.com/watch?v=2KhRmFHLuhE). And that’s clearly resonated with users — in the past year, Chromebook unit sales have grown 85% year over year (YOY),[^1] and PWA installs more than doubled.[^2]
+
+![Chromebook sales have increased 85% year over year and PWA installs have increased 175% year over year.](/images/posts/easy-to-build-monetize-and-discover-list-your-web-app-on-google-play/yoy_growth.png)
+
+With all the new capabilities being added to web apps — from saving to the local file system to device communication — we saw an opportunity to highlight the best experiences for Chromebook users.
+
+Search continues to be a quick and easy way to discover new web apps, but many people are also turning to app stores to find the best software for their device in one place. That’s why we gave developers the ability to list their PWA in Google Play on Chrome OS using [Trusted Web Activity](https://developers.google.com/web/android/trusted-web-activity). Now, users can discover web apps by browsing through collections and curated recommendations in the Play Store. And once they install it, they’ll be able to enjoy the full-screen app experience they love, powered by Chrome.
+
+Brands like [Kapwing](https://www.kapwing.com/), an online image, video, and GIF editing platform, have already seen impressive results after launching their PWA on the Play Store:
+
+![Quote from Julia Enthoven, CEO of Kapwing: "In the first five weeks of launching our PWA-ified website, the number of people creating videos through the PWA grew 36%. This growth in usage outpaced overall growth on the website, indicating higher retention among creators who installed the PWA."](/images/posts/easy-to-build-monetize-and-discover-list-your-web-app-on-google-play/kapwing_quote.png)
```

Comment:
If you want to match the quote styling that's already on chromeos.dev, you can replace this image with
```suggestion
> In the first five weeks of launching our PWA-ified website, the number of people creating videos through the PWA grew 36%. This growth in usage outpaced overall growth on the website, indicating higher retention among creators who installed the PWA.

Julia Enthoven, CEO of Kapwing {.cite}

```

Reply:
For a reference of what that'll look like: https://chromeos.dev/en/stories/concepts#results

Reply:
Thanks, I just pushed a change addressing this feedback